prometheus-client/src/main/java/ai/apptuit/metrics/client/prometheus/PrometheusClient.java

```diff
@@ -140,6 +140,12 @@ public QueryResponse query(long startEpochMillis, long endEpochMillis, String pr
     }
   }
 
+  static URL completeTrailingSlashURL(URL url) throws URISyntaxException, MalformedURLException {
+    String urlPath = url.toURI().toString();
+    urlPath = urlPath.endsWith("/") ? urlPath : (urlPath + "/");
```

Comment:
If endsWith /, return URL as is, instead of creating new URL

Reply:
done